internal/commands/app/create_test.go

```diff
@@ -565,18 +565,18 @@ func TestAppCreateCommandDisplay(t *testing.T) {
 				newAppInputs: newAppInputs{
 					Name:            "test-app",
 					Project:         "123",
-					From:            "from-app",
+					RemoteApp:       "remote-app",
 					Location:        realm.LocationIreland,
 					DeploymentModel: realm.DeploymentModelLocal,
 				},
-				Directory: "realm-app",
+				LocalPath: "realm-app",
 				Cluster:   "Cluster0",
 				DataLake:  "DataLake0",
 				DryRun:    true,
 			},
 		}
 		assert.Equal(t,
-			cli.Name+" app create --project 123 --name test-app --from from-app --app-dir realm-app --location IE --deployment-model LOCAL --cluster Cluster0 --data-lake DataLake0 --dry-run",
+			cli.Name+" app create --project 123 --name test-app --remote remote-app --app-dir realm-app --location IE --deployment-model LOCAL --cluster Cluster0 --data-lake DataLake0 --dry-run",
```

Comment:
this should probably be failing?  need to update `--app-dir` to `--local` (or change something else to make it fail first ;) )